src/plugins/profiling/server/routes/types.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export type StackTraceID = string;
+export type StackFrameID = string;
+export type FileID = string;
+
+export interface StackTraceEvent {
+  StackTraceID: StackTraceID;
+  Count: number;
+}
+
+export interface StackTrace {
+  FileID: string[];
+  FrameID: string[];
+  Type: string[];
+}
+
+export interface StackFrame {
+  FileName: string;
+  FunctionName: string;
+  FunctionOffset: number;
+  LineNumber: number;
+  SourceType: number;
+}
+
+export interface Executable {
+  FileName: string;
+}
```

Comment:
With this we are basically transposing the same structure of ES indices into the Kibana data model.
This can work for now, but I imagine at some point we will want to have dedicated types for the data model of how things will be _displayed_ rather than how they are stored.

Reply:
What we store and what we display are two different things. We likely will end up with two group of types, one to resemble the storage layout (or the fields that we query) and another to resemble the data to be displayed.